### PR TITLE
[#689] Reset cut is not updating view

### PR DESCRIPTION
**Pull request by emiliocortina**
https://github.com/HSF/phoenix/issues/689

### CODE REVIEW

File: packages/phoenix-event-display/src/managers/ui-manager/phoenix-menu/phoenix-menu-node.ts [hunk 1]
```diff
@@ -108,17 +108,13 @@ export class PhoenixMenuNode {
 
   /**
    * Add a config to the phoenix menu item.
-   * @param type Type of configuration.
-   * @param options Options for the config.
+   * @param config config to be displayed as a Phoenix Menu item.
    * @returns The current node.
    */
-  addConfig<T extends keyof PhoenixMenuConfigs>(
```
Comment (EdwardMoyse) [inline packages/phoenix-event-display/src/managers/ui-manager/phoenix-menu/phoenix-menu-node.ts:115]: So, the plan here is to avoid type inference and make it explicit in the caller? Is there an advantage to this i.e. was this the cause of the problem?

(Just for my understanding)

**Comment by EdwardMoyse** [comment]
Hey Emilio - thank you so much for doing this! I confirm that the reset cut button now works as expected. However, I think the cuts no longer work (for some reason which is not obvious).

For example, go to https://hepsoftwarefoundation.org/phoenix/#/atlas and load the attached [config file](https://github.com/user-attachments/files/17522438/run336567_evt2327102923-12.json) - the screen should go blank. But when I run this locally, the jets (the blue cones) are still visible.

Less importantly, you can find out how to fix the CI failure 
[here](https://github.com/HSF/phoenix/blob/main/guides/developers/test-setup.md#running-the-tests-locally)

TL;DR: run `yarn lint:fix`

File: packages/phoenix-event-display/src/managers/ui-manager/phoenix-menu/phoenix-menu-node.ts (hunk 1 above)
Comment (EdwardMoyse) [reply packages/phoenix-event-display/src/managers/ui-manager/phoenix-menu/phoenix-menu-node.ts:115]: ping @emiliocortina ?

**Comment by EdwardMoyse** [comment]
Hmm. A bit more debugging, and I think the issue might be that cut dialog ownership is not correct - cuts work for the first Jet collection ("AntiKt4EMTopoJets_xAOD") but for the second ("AntiKt4LCTopoJets_xAOD") the `collectionName` in `collectionFilter` is still "AntiKt4EMTopoJets_xAOD", AND if both are visible, the cuts from the second apply to the first.

**Comment by EdwardMoyse** [comment]
Yeah, this is definitely it. In `addObjectType` for the first collection there is no `configRangeSlider` on the cuts. However, once we get to the second collection there is. So in this case, the new configRangeSlider is never created here:
```typescript
 public getConfigRangeSlider(
    collectionFiltering: () => void,
  ): ConfigRangeSlider {
    if (this.configRangeSlider == undefined) {
      this.configRangeSlider = {
        type: 'rangeSlider',
```

**Comment by emiliocortina** [comment]
This is great! I took a look a couple of days ago and I found it very weird that it was working for the tracks, but not for the jets
